pkg/notifications/stream.go

```diff
@@ -0,0 +1,40 @@
+package notifications
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/stackrox/rox/generated/storage"
+)
+
+const bufferSize = 100
```

Comment:
nit: not sure whether this is overkill, but theoretically this could be an environment variable in case we need to increase the buffer (i.e. the rate of producing events is higher than consuming them). But, for now this is optional, we can address this later if required.